spring-boot-admin-server-ui/src/main/frontend/components/sba-input.vue

```diff
@@ -139,7 +151,7 @@ export default {
 
       const classNames = [];
 
-      if (this.error !== undefined) {
+      if (!!this.error) {
```

Comment:
Is this some fancy JS syntax I don't know? Or should we remove the double negation?

Reply:
Nah, this is not fancy. I have replaced it by an explicit check of null or undefined :)